static/js/components/UploadPhotometry.jsx

```diff
@@ -0,0 +1,382 @@
+                    Instrument
+                  </InputLabel>
+                  <Controller
+                    as={(
+                      <Select labelId="instrumentSelectLabel">
+                        <MenuItem value="multiple" key={0}>
+                          Multiple (requires instrument_id column below)
+                        </MenuItem>
+                        {
+                          instrumentList.map((instrument) => (
+                            <MenuItem value={instrument.id} key={instrument.id}>
+                              {`${instrument.name} (ID: ${instrument.id})`}
+                            </MenuItem>
+                          ))
+                        }
+                      </Select>
+                    )}
+                    name="instrumentID"
+                    rules={{ required: true }}
+                    control={control}
+                    defaultValue=""
+                  />
+                </FormControl>
+              </Box>
+            </Box>
+            <Box m={1}>
+              <HtmlTooltip
+                interactive
+                title={(
+                  <>
+                    <p>
+                      Use this form to upload flux- or mag-space photometry data
+                      (only one type per request, not mixed).
+                    </p>
+                    <p>
+                      Required fields (flux-space):&nbsp;
+                      <code>mjd,flux,fluxerr,zp,magsys,filter[,instrument_id]</code>
+                      <br />
+                      Required fields (mag-space):&nbsp;
+                      <code>mjd,mag,magerr,limiting_mag,magsys,filter[,instrument_id]</code>
+                      <br />
+                      See the&nbsp;
+                      <a href="https://skyportal.io/docs/api.html#/paths/~1api~1photometry/post">
+                        API docs
+                      </a>
+                      &nbsp;for other allowable fields (note: omit
+                      {' '}
+                      <code>obj_id</code>
+                      {' '}
+                      here).
+                    </p>
+                    <p>
+                      Other miscellanous metadata can be supplied by preceding the column
+                      name with
+                      {' '}
+                      <code>&quot;altdata.&quot;</code>
+                      {' '}
+                      (e.g.
+                      {' '}
+                      <code>&quot;altdata.metdata1&quot;</code>
+                      ).
+                      Such fields will ultimately be stored in the photometry table&apos;s
+                      {' '}
+                      <code>altdata</code>
+                      &nbsp;JSONB column, e.g.
+                      {' '}
+                      <code>
+                        {"{"}
+                        &quot;metadata1&quot;: &quot;value&quot;, ...
```

Comment:
Can we make this more concrete with an actual column that would be used in altdata, so people have a better sense of what to put here?

Reply:
Can you provide a realistic example?

Reply:
`{'calibrated_to': 'ps1', 'aperture_diameter_pixels':  6}`

Reply:
Done.